portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,82 @@
+            Everyone there is unique and I would'nt be where I am today without them. Along with this
+            oppertunity Howard has opened many doors for me to grow and learn.</p>
+      
+
+      <h2> What Projects Have I Worked On? </h2>
+        <ul>
+            <li> "Absent Future" - This project is my most recent. It is a first person shooter game
+                 about the horrors of a future affected by global warming. This was made my friends
+                 and I during an Amazon Games hackathon using Lumberyard, Amazon's game engine.
+                 My specific role was the back end developer, I was responsible for making the
+                 computer generated AI enemies run towards the player and if one touched the player,
+                 one life would be lost. We ended up winning the grand prize with this project.
+                 This was the first hackathon we ever won, so this project is near and dear to my
+                 heart. </li>
+
+            <li> "Naturally U"- This project is something I'm proud of because it's all about natural hair,
+                  something that I love about myself and know a lot about. Naturally U is a all-in-one resource for 
+                  natural hair.It shows you affordable hair products for your hair type, style tutorials, and 
+                  the closest barbers or hair stylists to the user. Naturally U was made during the UNCF CS Summer
+                  Academy where I worked with a team on 3 to complete this project using HTML,CSS,and Python.My specific role was the full 
+                  stack developer. I worked on the front end of several pages of the app including the home page and the tutorial selection
+                  page and on the back end of the login page. At the end of the summer pitch
+                  competition our project took the grand prize!</li>
+
+            <li> "Little Green Foot"- This project is a sort of environmental effect tracker. In this app
+                  you could mark "environmentally friendly" actions you did that day such as carpooling to work, recycling, etc.
+                  and you would earn points from those actions which the user could cash out those points in the
+                  form of gift cards and other prizes. My role on this project was the full stack developer. I worked 
+                  with the front end developer to create all the visuals for the mock up and I worked with the backend developer
+                  to create the point system algorithm.</l1>
+        </ul>
+        <h2> If You Could Travel Anywhere in the World,Where Would You Go Kayla? </h2>
+            <p> Wow,thank y'all for asking!</p>
+
+            <p> I would say Italy is my number 1, I've always wanted to go there but if I
+                had to choose somewhere else...</p>
+            <img src="/images/italy.jpg" alt=" a cool and free pic of Italy!" style="width:50% ;height:50%">
```

Comment:
Optional: It looks like all of these `img` tags employ the same `style` values. Consider defining a CSS `class` for this instead for easy reusability and upkeep. i.e. if they all share a class, then if you decide you ever want width to be "80%" instead of "50%" then you only have to change it in one place!